lib/matplotlib/bezier.py

```diff
@@ -68,6 +68,54 @@ def get_normal_points(cx, cy, cos_t, sin_t, length):
     return x1, y1, x2, y2
 
 
+# deprecated routines (moved to `.Path`)
+
+
+@cbook.deprecated("3.2")
+def inside_circle(cx, cy, r):
```

Comment:
As for split_path_inout I would just leave this as public API reexported into this module, for now.

Reply:
Public API where? Currently I've moved it into `patches.py`, which is the only place where it is used. I hesitate to make `inside_circle` a part of the public API of that module, since it's such a non-public-facing part of the module's internals.

In addition, we can't REALLY just re-exporting it into this module from there, since `patches.py` already does `from .bezier import ...`. I fixed this now so it doesn't explicitly repeat the code, which I figure is better so that the implementations don't diverge.

Reply:
As above, you can patch bezier.py to add this function to it at the end of patches.py?  In fact it can stay private in patches, and be "public" in bezier (`bezier.inside_circle = _inside_circle` at the end of patches.py).

Reply:
but then the API will only remain unchanged as long as the user imports patches. If they import bezier only, the patch will not be applied and the functions will be missing?